src/gsy_e/setup/strategy_tests/external_devices_send_forecasts.py

```diff
@@ -0,0 +1,67 @@
+# pylint: disable=missing-docstring
+import os
+import platform
+
+from gsy_framework.constants_limits import ConstSettings
+
+from gsy_e.models.area import Market, Asset
+from gsy_e.models.strategy.external_strategies.load import LoadForecastExternalStrategy
+from gsy_e.models.strategy.external_strategies.pv import PVForecastExternalStrategy
+from gsy_e.models.strategy.load_hours import LoadHoursStrategy
+from gsy_e.models.strategy.storage import StorageStrategy
+
+current_dir = os.path.dirname(__file__)
+print(current_dir)
+print(platform.python_implementation())
```

Comment:
Is there an intended reason for these to reside here?

Reply:
Actually not. I copied an already existing strategy and changed what I needed, so I think is just a copy paste thing. I've removed it anyway. Thanks